mobile-widgets/qml/TemplateComboBox.qml

```diff
@@ -5,5 +5,7 @@ import QtQuick.Layouts 1.11
 
 ComboBox {
 	Layout.fillWidth: true
+	Layout.preferredHeight: Kirigami.Units.gridUnit * 2.5
+	inputMethodHints: Qt.ImhNoPredictiveText
```

Comment:
do we always want that height? and do we always want no predictive text?
I don't know the answer, just asking to get a bit more background from you...

Reply:
This should be the standard height.
And yes we want default to be no predictive text.

But please remember these properties are not made of stone, they can be overwritten in every instance. The idea is simply to add the default use here, so we type less in each instance.